Marlin/Marlin_main.cpp

```diff
@@ -5130,9 +5130,9 @@ inline void gcode_M252() {
         servo[servo_index].move(servo_position);
       else {
         SERIAL_ECHO_START;
-        SERIAL_ECHO("Servo ");
+        SERIAL_PROTOCOLPGM("Servo ");
```

Comment:
These are unrelated to the regulator, just putting these strings in prog mem right?